run-until-success-with-backoff

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -euo pipefail
+
+set -x
```

Comment:
Maybe we should leave this out to avoid more output in the test, and let this helper print useful messages.

Reply:
Done.